package.json

```diff
@@ -18,8 +18,10 @@
     "@types/node": "^8.0.20",
     "@types/react": "^16.0.2",
     "@types/react-dom": "^15.5.2",
+    "@types/react-router-dom": "^4.0.7",
```

Comment:
I don't really have an argument either way. I come from Angular 2 which is TS all the way through so I'm just used to it. We could check out Flow for this project though. I've heard it's better at inference and doesn't require additional type libraries.

I'm not really concerned with types becoming out of date with the library though. Unless we're always on the edge version of packages, Definitely Typed keeps up to date pretty well.